### ISSUE TITLE
Windows build fails at joyconfig

### ISSUE THREAD

**Issue by bbyam** [libretro/RetroArch#2755]
I'm a bit confused by this.  I'm trying to build with the prepackaged MinGW toolchain, downloaded from here: themaister.net/retroarch-dl/MinGW-win64-RetroArch-dev-winpthread.7z

The latest Makefile.win attempts to build retroarch-joyconfig.exe, but there don't appear to be any objects to build.  Here's what I see in the latest Makefile.win:

At the top:
`JTARGET = tools/retroarch-joyconfig.exe`

The make all command:
`all: $(TARGET) $(JTARGET)`

However, there are no object files files for it:

    JOYCONFIG_OBJ    :=
    JOYCONFIG_LIBS   :=
    RARCH_JOYCONFIG_OBJ := $(addprefix $(OBJDIR)/,$(JOYCONFIG_OBJ))

And so, the line that tries to link it is failing:
`$(Q)$(CC) -o $@ $(RARCH_JOYCONFIG_OBJ) -ldxguid -ldinput8 -lole32 $(LDFLAGS) $(JOYCONFIG_LIBS)`

I echoed this line out during the make, and this is what it shows:
> gcc -o tools/retroarch-joyconfig.exe  -ldxguid -ldinput8 -lole32 -L. -static-libgcc

Thus, it is failing because it can't find a main.  What do I need to do to get this to build correctly?  I couldn't find what object files are needed in any of the makefiles.

**Comment by twinaphex** [comment]
retroarch-joyconfig has been gone for a long time.

Also, that toolchain is no longer recommended to be used.

**Comment by bbyam** [comment]
Ah, that explains it - I'm using an old toolchain.

I'll try with this:
https://github.com/libretro/RetroArch/wiki/Compilation-guide-%28Windows%29

**Comment by bbyam** [comment]
So, I used the compilation guide I linked above, and still got the exact same error.  Since joyconfig is supposedly gone, I just removed it from Makefile.win.  I also had to add HAVE_NETWORK_GAMEPAD = 1 to the makefile in order to get it to build without a different link error, missing remote.o.

With those 2 changes, the instructions in that Compilation guide now work (aka, compile via "mingw32-make -f Makefile.win")
